ui/src/components/nested-fields/QasNestedFields.vue

```diff
@@ -356,6 +356,10 @@ export default {
     getDefaultActionsMenuList (index, row) {
       const list = {}
 
+      const destroyProps = typeof this.buttonDestroyProps === 'function'
```

Comment:
essa validação não poderia ser feita fora da função para não ter que ser executada toda vez que o método for chamado? sendo assim só executaria 1x, poderia ser uma computada neste caso.